main.py

```diff
@@ -102,7 +105,13 @@ def discover_delete_images(region_name, clusters):
     for image in running_containers:
         print(image)
 
+    ignore_repo_regex = re.compile(IGNORE_REPO_REGEX)
+
     for repository in repositories:
+      if re.search(ignore_repo_regex, repository['repositoryUri']):
+        print("------------------------")
+        print("Skipping repository :" + repository['repositoryUri'])
+      else:
```

Comment:
Do not mix the 2-space tab and 4 spaces. Instead of else section use continue as you did before